pkgs/build-support/rust/fetch-cargo-tarball/default.nix

```diff
@@ -73,6 +73,10 @@ in stdenv.mkDerivation ({
 
     ${cargoUpdateHook}
 
+    # Override the `http.cainfo` option usually specified in `.cargo/config`.
+    export CARGO_HTTP_CAINFO=${cacert}/etc/ssl/certs/ca-bundle.crt
+    export SSL_CERT_FILE=${cacert}/etc/ssl/certs/ca-bundle.crt
```

Comment:
`SSL_CERT_FILE` is set by the `cacert` `setupHook` so probably don't need this.
```suggestion
```

Reply:
Good catch, thanks!